app/code/community/Yotpo/Yotpo/sql/yotpo_setup/mysql4-install-1.6.4.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+$installer = $this;
+/* @var $installer Mage_Core_Model_Resource_Setup */
+
+Mage::log('mysql4-install-1.6.4.php');
+
+$installer->startSetup();
+
+$prefix = (string) Mage::getConfig()->getTablePrefix();
+Mage::log("prefix - " . $prefix);
+
+$base_table = "yotpo_rich_snippets";
+$table_with_quotes = "`" . $base_table .  "`";
+
+if ($prefix != "") {
+    $base_table = $prefix . $base_table;
+    $table_with_quotes =  "`". $base_table . "`";
```

Comment:
i don't like the name $table_with_quotes so much...
consider changing just  to table_name (your call)